examples/file-share-svelte/src/lib/components/FileItem.svelte

```diff
@@ -66,14 +55,16 @@
   class="flex items-center justify-between rounded-lg border border-gray-200 bg-white p-4"
   transition:slide={{ duration: 200 }}
 >
-  <div class="flex items-center space-x-4">
+  <div class="flex items-center space-x-4 flex-grow">
     <div class="flex h-10 w-10 items-center justify-center rounded-lg bg-blue-100 text-blue-600">
       <File class="h-6 w-6" />
     </div>
-    <div>
-      <a href="/file/{file.id}" class="hover:text-blue-600 hover:underline">
+    <div class="flex-grow">
+      {#if isAdmin}
+        <input  class="font-medium text-gray-900 w-full py-1" type="text" bind:value={file.name} />
```

Comment:
Svelte complains here:

```
[svelte] binding_property_non_reactive`bind:value={file.name}` (src/lib/components/FileItem.svelte:64:73) is binding to a non-reactive propertyhttps://svelte.dev/e/binding_property_non_reactive
```

but the reactivity works, because it's powered by Jazz instead of Svelte (it's not fine-grained thou, probably)